gradle/libs.versions.toml

```diff
@@ -40,3 +40,4 @@ teku = "23.1.1"
 tuweni = "2.4.2"
 vertx = "4.5.14"
 web3j = "4.12.2"
+kotlin = "2.0.0"
```

Comment:
This I don't think is needed, as `libs.versions.kotlin.get()` is not used in anywhere